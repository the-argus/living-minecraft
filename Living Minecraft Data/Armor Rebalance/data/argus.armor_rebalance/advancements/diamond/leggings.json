{
  "criteria": {
    "requirement": {
      "trigger": "minecraft:recipe_unlocked",
      "conditions": {
        "recipe": "minecraft:diamond_leggings"
      }
    }
  },
  "rewards": {
    "function": "argus.armor_rebalance:diamond/leggings",
    "loot": [
      "argus.armor_rebalance:diamond/leggings"
    ]
  }
}
