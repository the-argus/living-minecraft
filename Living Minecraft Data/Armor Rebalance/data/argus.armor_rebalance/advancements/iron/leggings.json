{
  "criteria": {
    "requirement": {
      "trigger": "minecraft:recipe_unlocked",
      "conditions": {
        "recipe": "minecraft:iron_leggings"
      }
    }
  },
  "rewards": {
    "function": "argus.armor_rebalance:condition/iron/leggings"
  }
}
