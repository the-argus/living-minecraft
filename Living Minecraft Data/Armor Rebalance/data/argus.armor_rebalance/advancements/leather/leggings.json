{
  "criteria": {
    "requirement": {
      "trigger": "minecraft:recipe_unlocked",
      "conditions": {
        "recipe": "minecraft:leather_leggings"
      }
    }
  },
  "rewards": {
    "function": "argus.armor_rebalance:leather/leggings",
    "loot": [
      "argus.armor_rebalance:leather/leggings"
    ]
  }
}
