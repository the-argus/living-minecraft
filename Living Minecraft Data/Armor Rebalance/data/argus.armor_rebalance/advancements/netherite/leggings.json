{
  "criteria": {
    "requirement": {
      "trigger": "minecraft:recipe_unlocked",
      "conditions": {
        "recipe": "argus.armor_rebalance:netherite_leggings_shapeless"
      }
    }
  },
  "rewards": {
    "function": "argus.armor_rebalance:condition/netherite/leggings"
  }
}
