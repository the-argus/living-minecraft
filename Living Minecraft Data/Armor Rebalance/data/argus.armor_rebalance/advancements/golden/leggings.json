{
  "criteria": {
    "requirement": {
      "trigger": "minecraft:recipe_unlocked",
      "conditions": {
        "recipe": "minecraft:golden_leggings"
      }
    }
  },
  "rewards": {
    "function": "argus.armor_rebalance:golden/leggings",
    "loot": [
      "argus.armor_rebalance:golden/leggings"
    ]
  }
}
